src/coverlet.collector/Utilities/CoverletConstants.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Coverlet.Collector.Utilities
+{
+    internal static class CoverletConstants
+    {
+        public const string FriendlyName = "XPlat code coverage";
+        public const string DefaultUri = @"datacollector://Microsoft/CoverletCodeCoverage/1.0";
+        public const string DataCollectorName = "CoverletCoverageDataCollector";
+        public const string DefaultReportFormat = "cobertura";
+        public const string DefaultFileName = "coverage";
+        public const string IncludeFiltersElementName = "Include";
+        public const string IncludeDirectoriesElementName = "IncludeDirectory";
+        public const string ExcludeFiltersElementName = "Exclude";
+        public const string ExcludeSourceFilesElementName = "ExcludeByFile";
+        public const string ExcludeAttributesElementName = "ExcludeByAttribute";
+        public const string MergeWithElementName = "MergeWith";
+        public const string UseSourceLinkElementName = "UseSourceLink";
+        public const string SingleHitElementName = "SingleHit";
+        public const string IncludeTestAssemblyElementName = "IncludeTestAssembly";
+        public const string TestSourcesPropertyName = "TestSources";
+        public const string ReportFormatElementName = "Format";
+        public const string DefaultExcludeFilter = "[coverlet.*]*";
```

Comment:
In case of `dotnet vstest`, all dlls and their dependencies are copied to a publish folder by `dotnet publish`. So test dlls and coverlet.collector resides in the same folder, hence when running coverage via `dotnet vstest unittest.dll --collect:"Xplat code coverage"`, it instruments coverlet.collector also, and tries to copy it here [https://github.com/tonerdo/coverlet/blob/f64337e74ef605b3ad3d9a7d304e02274c4383a6/src/coverlet.core/Helpers/InstrumentationHelper.cs#L125](https://github.com/tonerdo/coverlet/blob/f64337e74ef605b3ad3d9a7d304e02274c4383a6/src/coverlet.core/Helpers/InstrumentationHelper.cs#L125), which throws error saying the file is already in use by some other process. That's why I white listed the coverlet.* dlls from getting instrumented by default.

Reply:
Thanks